### PR TITLE
Rework quick start and examples section

### PR DESCRIPTION
**Pull request by steventux**
https://trello.com/c/vzsT6fUH/1074-write-quick-start-guide-and-examples-section-of-api-documentation-3

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ It's useful to have some understanding of how RESTful APIs use HTTP verbs such a
 
 ## Quick start guide and examples
 
-The GOV.UK Content API can be interrogated using the command line utilities `curl` or `wget`. These provide a quick way of seeing the JSON API responses prior to developing more complex integrations.
+The GOV.UK Content API can be interrogated using the command line utilities `curl` or `wget`. These provide a quick way of viewing JSON API responses prior to developing more complex integrations.
```
Comment (kevindew) [inline source/index.html.md]: interrogated? sounds quite draconian hehe

Comment (steventux) [reply source/index.html.md]: Agree it's a bit _big brother_ for government documentation, will use a less invasive term.

Comment (kevindew) [reply source/index.html.md]: Sure - I'm having a bit of a ponder about to rework this section a bit to be a sequence of quick steps (just as a warning)

File: source/index.html.md [hunk 2]
```diff
@@ -36,106 +36,64 @@ Run the following command in a terminal emulator to see a live JSON response fro
...
+    ```
+
+  3. The content itself:
+
+    ```json
+      "details": {
+        "body" : "\n<div class=\"highlight-answer\">\n<p>The standard <abbr title=\"Value Added Tax\">VAT</abbr> rate is <em>20%</em></p>\n</div>..."
+        ...
+      }
+    ```
```
Comment (kevindew) [inline source/index.html.md]: This information looks really useful but I think it probably belongs in reference documentation.

**Comment by kevindew** [comment]
I'm wondering if something like this is sufficient for this section and other bits should be more referency:

===

You can use this API to look up the data that is used to power content on GOV.UK to get started with this:

1. Pick a page on GOV.UK eg. https://www.gov.uk/take-pet-abroad
2. Make a note of the path eg. “/take-pet-abroad”
3. Using a tool such as [curl](#) or [Postman](#) make a GET request to https://www.gov.uk/api/content/take-pet-abroad 
4. You’ll receive a JSON response, the fields for this are explained in the [reference documentation](#)

For example, using curl and the [jq](https://stedolan.github.io/jq/) utility:

`curl https://www.gov.uk/api/content/take-pet-abroad | jq`

**Comment by steventux** [comment]
@kevindew :+1: I've amended as described and added the relevant links

**Pull request merged**